pkgs/development/tools/gauge/default.nix

```diff
@@ -17,9 +23,8 @@ buildGoModule rec {
 
   meta = with lib; {
     description = "Light weight cross-platform test automation";
-    mainProgram = "gauge";
```

Comment:
Why should we drop this?

Reply:
we shouldn't, i messed something up while rebasing